api/api_handler.go

```diff
@@ -27,7 +27,7 @@ func (a *ApiHandler) GetAddressDags(ctx echo.Context, address AddressFilter, par
 		return ctx.JSON(http.StatusNotFound, "Data for address "+address+" not found")
 	}
 
-	ret, pagination, err := storage.GetObjectsPage[Dag](a.storage, address, uint64(params.Pagination.Start), params.Pagination.Limit)
+	ret, pagination, err := storage.GetObjectsPage[Dag](a.storage, address, getPaginationStart(params.Pagination.Start), params.Pagination.Limit)
```

Comment:
If you making Pagination optional you should also check if it is nil or not before you are trying to get something from it